public/css/screen.styl

```diff
@@ -203,6 +203,189 @@ a.dropdown-toggle {
 .event-navbar .admin-button {
   margin-top: 15px;
 }
+/* ================== EMAIL PREVIEW MODE STYLES ==================== */
+
+.email-preview-container {
+  padding: 0px;
+  margin-top: 0px;  
+  font-family: Helvetica,Arial,sans-serif;
+
+  .col-lg-12 {
+    padding: 0px;
+  }
+
+  .email-preview-fields-container {
+    border 2px solid #f2f2f2;
+    background-color: #f2f2f2;
+    margin-top: 10px;
+  }
+
+  .control-label {
+    color: #999;
+    font-weight: 400;
+    font-size: 14px;
+    margin-top: 5px;
+  }
+
+  .field-input {
+    color: #444;
+    font-weight: 400;
+    font-size: 14px;
+    margin-top: 5px;
+  }
+
+  .form-group {
+    margin-top: 30px;
+    margin-left: 25px;
+    margin-right: 40px;
+    padding: 0px;
+  }
+
+  .email-header {
+    padding: 15px;
+    padding-bottom: 0px;
+    margin-bottom: 10px;  
+  }
+
+  .email-footer {
+    padding: 15px;
+    margin-top: 10px;
+  }
+
+  .breakout-info {
+    margin-top: 10px;
+    padding: 15px;
+  }
+
+  .email-to {
+    background-color: white;
+    padding: 2px;
+    padding-left: 10px;
+    margin-bottom: 5px;
+  }
+
+  .email-from {
+    background-color: white;
+    padding: 2px;
+    padding-left: 10px;
+    margin-bottom: 5px;
+  }
+
+  .email-sub {
+    background-color: white;
+    padding: 2px;
+    padding-left: 10px;
+    margin-bottom: 5px;
+  }
+
+  .nav-button-row {
+    padding: 0px;
+   
+    .col-lg-3 {
+      padding: 0px;
+    }
+
+    p {
+      margin-top: 15px;
+      font-size: 16px;
+    }
+
+    .col-lg-1 {
+      padding: 0px;
+      padding-right: 0px;
+      padding-left: 5px;
+    }
+
+    button {
+      width: 100%;
+      margin: 0px;
+      margin-top: 10px;
+      margin-bottom: 40px;
+      font-size: 16px;
+      letter-spacing: 0.07rem;
+    }
+
+    .previous {
+      margin-right: 2px;
+
+      .fa {
```

Comment:
Same comment from bootstrap above.  Add a new class ``white-text`` or some such rather than altering the definition of ``fa``.